src/snapshot.c

```diff
@@ -367,7 +354,7 @@ RRStatus initiateSnapshot(RedisRaftCtx *rr)
     rr->curr_snapshot_last_idx = rr->snapshot_info.last_applied_idx;
     rr->curr_snapshot_last_term = rr->snapshot_info.last_applied_term;
     rr->snapshot_in_progress = true;
-    rr->curr_snapshot_start_time = RedisModule_Milliseconds();
+    rr->curr_snapshot_start_time = RedisModule_MonotonicMicroseconds();
```

Comment:
moved to monotonic timestamp, just to prevent possible negative diff on wall clock adjustments